cmake.deps/cmake/BuildMsgpack.cmake

```diff
@@ -1,23 +1,16 @@
 set(MSGPACK_CMAKE_ARGS
-  -DMSGPACK_BUILD_TESTS=OFF
-  -DMSGPACK_BUILD_EXAMPLES=OFF
-  -DCMAKE_INSTALL_PREFIX=${DEPS_INSTALL_DIR}
-  -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
-  -DCMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES_ALT_SEP}
-  "-DCMAKE_C_FLAGS:STRING=-fPIC"
-  -DCMAKE_GENERATOR=${CMAKE_GENERATOR})
-
-if(MSVC)
-  set(MSGPACK_CMAKE_ARGS
     -DMSGPACK_BUILD_TESTS=OFF
     -DMSGPACK_BUILD_EXAMPLES=OFF
     -DCMAKE_INSTALL_PREFIX=${DEPS_INSTALL_DIR}
     -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+    -DCMAKE_GENERATOR=${CMAKE_GENERATOR}
     -DCMAKE_GENERATOR_PLATFORM=${CMAKE_GENERATOR_PLATFORM}
-    ${BUILD_TYPE_STRING}
-    # Make sure we use the same generator, otherwise we may
-    # accidentally end up using different MSVC runtimes
-    -DCMAKE_GENERATOR=${CMAKE_GENERATOR})
+    ${BUILD_TYPE_STRING})
+
+if(NOT MSVC)
+  list(APPEND MSGPACK_CMAKE_ARGS
+    -DCMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES_ALT_SEP}
```

Comment:
```suggestion
    "-DCMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES_ALT_SEP}"
```

Reply:
?

Reply:
it's just a good habit when dealing with cmake to use quotes

Reply:
I'll take the risk if it means less cluttered cmake code ;)